### DISCUSSION TITLE
Lift-independent drag coefficient attributable to flaps

### DISCUSSION THREAD

**Discussion by TheAviationDoctor** [junzis/openap#38] [Q&A]
Hi @junzis,

_Note: this post initially contained two questions, until I decided to move the second question to [its own post](https://github.com/junzis/openap/discussions/39)._

OpenAP is amazing work. I have a question related to the [drag polar](https://github.com/junzis/openap/blob/master/openap/drag.py) script, please.

Table 3 and equations 24-25 of [Sun et al. (2020)](https://junzis.com/files/openap_dragpolar.pdf) refer to `δf` as the flap deflection in degrees. The `drag.py` script also calls for the `flap_angle` argument to be passed in degrees.

However, the script then converts degrees to radians in the last line of the `delta_cd_flap` calculation:

```
delta_cd_flap = (
            lambda_f
            * (cfc) ** 1.38
            * (SfS)
            * self.np.sin(flap_angle * self.np.pi / 180) ** 2
        )
```

**I just wanted to confirm that this is intentional**, since equations 24-25 do not reflect the conversion to radian ( `δf * π / 180`). Assuming a flap deflection angle of 10°, and using `Cf/C` and `Sf/S` values for the [A320neo](https://github.com/junzis/openap/blob/master/openap/data/dragpolar/a20n.yml) (and using a `λf` of .9 for slotted flaps), I get a value of .0004 if converting `δf` to radians, which looks small for the drag generated by 10° flaps, and an order of magnitude smaller than if I leave `δf` in degrees (.004).

Thank you in advance, @junzis.

**Comment by junzis** [comment]
@TheAviationDoctor, thanks for the comment.
That's correct. For scientific notation, the angles are usually written in degrees for the better understanding. For computers, it always takes the radians as input.
